app.py

```diff
@@ -268,17 +258,17 @@ def start_convert(self):
     app = DocToCSV()
     app.title("The Brewery Pioneers: CSV Import")
 
-    w = 600 
-    h = 600 
+    w = 400
+    h = 300
```

Comment:
Need to increase the height to at least 350. Otherwise, progress bar is hidden in the ConvertFile frame.

Reply:
oops.jpg 
Will revise!